src/utils/trimLine.js

```diff
@@ -1,6 +1,7 @@
 'use strict'
 
-var urlOrContentRe = /(["'].+["'])|( +|:)url\(.+\)/
+var contentRe = /(content:\s*["'])(?:.+)(["'])/
```

Comment:
This regexp creates 2 match groups. The first is the `content: "` part and the second basically just amounts to the closing quotation mark and possibly semicolon.